src/_igraph/igraphmodule.c

```diff
@@ -840,6 +840,9 @@ extern PyObject* igraphmodule_arpack_options_default;
   PyModule_AddIntConstant(m, "TRANSITIVITY_NAN", IGRAPH_TRANSITIVITY_NAN);
   PyModule_AddIntConstant(m, "TRANSITIVITY_ZERO", IGRAPH_TRANSITIVITY_ZERO);
 
+  PyModule_AddIntConstant(m, "CPM", 0);
```

Comment:
You might not even need these - just use strings on the Python side (i.e. `"cpm"` and `"modularity"`). This would be consistent with the enums in the C core where we also allow string representations in the Python layer - these are converted on-the-fly to the appropriate integers (see the conversion functions in `convert.c`).

Reply:
I'm not entirely sure if I understand what you mean here. To be sure, the only place I actually use this constant is here:
https://github.com/igraph/python-igraph/blob/c6f6c1d01e0564077177d2cec5d49d5e8da0cf00/src/igraph/__init__.py#L1428-L1435
In other words, it is not a proper enum in the `igraph` C core, it is just something that I use in Python. Do you prefer to use strings instead?

Reply:
Yes, please, just use a single string instead if that's okay with you. I'd rather not add arbitrary integer aliases to the `igraph._igraph` module any more; the output of `dir(igraph._igraph)` is polluted enough as it is.